docs/basics/Gotchas/RedHat.md

```diff
@@ -0,0 +1,58 @@
+# RedHat / CentOS / Fedora
+
+## RedHat/CentOS vs Fedora
```

Comment:
This doesn't really sound like a "gotcha" to me? I wonder if this would be better placed as its own page about the differences between the 2 (3) distros

Reply:
Agreed. I feel this whole PR would be better as a distro informational break down page rather that gotchas as well. Like what's here though!